src/Arcus.Templates.WebApi/Startup.cs

```diff
@@ -247,17 +247,21 @@ private static void RestrictToJsonContentType(MvcOptions options)
             options.OutputFormatters.RemoveType<StringOutputFormatter>();
         }
 
-        private static void AddEnumAsStringRepresentation(MvcOptions options)
+        private static void ConfigureJsonFormatters(MvcOptions options)
         {
             var onlyJsonInputFormatters = options.InputFormatters.OfType<SystemTextJsonInputFormatter>();
             foreach (SystemTextJsonInputFormatter inputFormatter in onlyJsonInputFormatters)
             {
+                inputFormatter.SerializerOptions.MaxDepth = 10;
```

Comment:
Should we do this?

Reply:
I'm more of the opinion to have this in place and that the developer change it to something higher if they need it, then to leave it open to 64 by default and maybe overload something.

Reply:
WDYT @fgheysels ?

Reply:
In my humble opinion, we should stick to the defaults defined in .NET.
Target audience is developers, so we shouldn't be too restrictive or 'parental' (or whatever you call it :) ).

Reply:
Frankly, I was thinking the same on this one

Reply:
Ok, then should we also remove it in the Azure Function HTTP trigger 'cause it's defined there too. I can do that in this PR too.

Reply:
Left the `IgnoreNullValues` still there, but removed the `MaxDepth` in both the web API as the Azure Functions HTTP trigger template. Resolving.